docker/alias/alias_computation.py

```diff
@@ -0,0 +1,138 @@
+#!/usr/bin/env python3
+# Copyright 2023 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""OSV alias computation."""
+import datetime
+import logging
+
+from google.cloud import ndb
+
+import osv
+import osv.logs
+
+ALIAS_GROUP_VULN_LIMIT = 16
+VULN_ALIASES_LIMIT = 5
+
+bugs_map = {}
+aliases_map = {}
```

Comment:
Can we avoid these global variables?

Reply:
Also, can we rename these to be more descriptive with explanatory comments? i.e. 

```
# Mapping of ID to direct aliases for that bug. 
bug_direct_aliases = {}

# Reverse map of aliases to the bug. 
aliases_to_bug = {}
```